docs/src/content/en/docs/deployment/cloud-providers/amazon-ec2.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "Amazon EC2"
+description: "Deploy your Mastra applications to Amazon EC2."
+---
+
+import { Callout, Steps } from "nextra/components";
+import ServerConfig from "@/components/content-blocks/server-config.mdx";
+
+## Amazon EC2
+
+Deploy your Mastra applications to Amazon EC2 (Elastic Cloud Compute).
+
+<Callout>
+  This guide assumes your Mastra application has been created using the default
+  `npx create-mastra@latest` command.
+  For more information on how to create a new Mastra application,
+  refer to our [getting started guide](./../../getting-started/installation.mdx)
+</Callout>
+
+### Setting up EC2
+
+<Steps>
+
+#### Log into your AWS console
+
+Navigate to the [AWS Management Console](https://aws.amazon.com/console/) and sign in to your account.
+
+#### Navigate to EC2
+
+Head over to **All services** on your left panel. Under **Compute**, click on **EC2**.
```

Comment:
Consider revising this: "on your left panel". 
...in the left-hand side navigation, 
or 
... side bar navigation
perhaps?